pyproject.toml

```diff
@@ -78,7 +79,7 @@ tests = [
     "tox",
```

Comment:
Why was boto3 removed from the `tests` extra? Isn't it required to run the tests?

Reply:
This is right but asyncpg is also required for the tests and it is not in the tests section

Reply:
But I see that the test will be skipped if asyncpg is not installed

Reply:
It is not required in the default configuration so that tox can run with or without it.

Reply:
Yeah we just figured it out